docs/deploy-from-precompiled-binaries-windows.md

```diff
@@ -0,0 +1,107 @@
+
+[Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/sql-server-2017-editions)
+
+## Downloading the precomplied binaries
+
+* Navigate to the <a href="https://github.com/VirtoCommerce/vc-platform/releases">Releases section of Virto Commerce Platform in GitHub.</a>
+
+* You will find **VirtoCommerce.Platform.3.x.x.zip** file. In this file the site has already been built and can be run without additional compilation. The source code is not included.
+
+* Unpack this zip to a local directory **C:\vc-platform-3**. After that you will have the directory with Platform precompiled files.
+
+## Setup
+
+### Configure application strings
+
+* Open the **appsettings.json** file in a text editor.
+* In the **ConnectionStrings** section change **VirtoCommerce** node (provided user should have permission to create new database):
+
+```json
+    "ConnectionStrings": {
+        "VirtoCommerce" : "Data Source={SQL Server URL};Initial Catalog={Database name};Persist Security Info=True;User ID={User name};Password={User password};MultipleActiveResultSets=True;Connect Timeout=30"
+    },
+
+```
+
+* In the **Assets** section set public url for assets `Assets:FileSystem:PublicUrl` with url of your application, this step is needed in order for display images
+
+```json
+"Assets": {
+        "Provider": "FileSystem",
+        "FileSystem": {
+            "RootPath": "~/assets",
+            "PublicUrl": "https://localhost:5001/assets/" <-- Set your platform application url with port localhost:5001
+        },
+    },
+```
+
+### Running the Platform by CLI "dotnet"
+
+* Install and trust HTTPS certificate
+
+Run to trust the .NET Core SDK HTTPS development certificate:
+
+```console
+dotnet.exe dev-certs https --trust
+```
+
+Read more about [enforcing HTTPS in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-3.0&tabs=visual-studio#trust)
+
+* Run the Platform by following command:
+
+```console
+dotnet.exe C:\vc-platform-3\VirtoCommerce.Platform.Web.dll
+```
+
+The output in the console will say something like:
+
+```console
+Now listening on: http://localhost:5000
+Now listening on: https://localhost:5001
+```
+
+### First run sign in
+
+* Open `https://localhost:5001` url in your browser. "Your connection is not private" might appear. Click "Advanced" and "Proceed to ...".
+Read more on removing this error and using a self-signed certificate: [Trust the ASP.NET Core HTTPS development certificate](https://www.hanselman.com/blog/DevelopingLocallyWithASPNETCoreUnderHTTPSSSLAndSelfSignedCerts.aspx)
+* The application will create and initialize database on the first request. After that you should see the sign in page. Use the following credentials:
+  * Login: **admin**
+  * Password: **store**
+
+### Host on Windows with IIS
+
+VirtoCommerce.Platform project already include the **web.config** ffile is already included in the release package. It contains all the necessary settings for running in IIS.
```

Comment:
The required **web.config** file was already included in the release package.

Reply:
fixed